### DISCUSSION TITLE
Address override not including ZIP

### DISCUSSION THREAD

**Discussion by landolfi-us** [code4recovery/12-step-meeting-list#1188] [General]
`Hi,`

My functions.php has this address override:
`
if (function_exists('tsml_custom_addresses')) {
    tsml_custom_addresses(array(
        '466 W Illinois Route 173' => array(
            'formatted_address' => '466 W Route 173, Antioch, IL 60002',
            'city' => 'Antioch',
            'latitude' => 42.4740932,
            'longitude' => -88.1033171,
            'approximate' => 'no',
        ),
    ));
}
`
But the meeting in the online meeting directory and the meeting JSON feed do not include the ZIP, which is causing Meeting Guide to use the Google geocoder, and Google geocoder placing the meeting in the wrong city is the original reason for the address override. Is the ZIP formatted correctly in the above override? Is this a bug?

**Comment by landolfi-us** [comment]
Never mind! I had to edit one of the meetings and apply to all at that location to get it to refresh, just updating functions.php did not automatically update the address.

**Reply by joshreisner** [reply]
typically the formatted_address we get back from Google includes the country, eg `466 W Route 173, Antioch, IL 60002, USA` not sure if this makes a difference for you or not
